modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -271,7 +271,6 @@ protected function register_controls() {
 				],
 				'default' => 'div',
 				'separator' => 'before',
-				'render_type' => 'template',
```

Comment:
@Barcov Not sure this is correct? This will cause the HTML tag to not change in the Editor. The data bindings should fix the issue. WDYT?

Reply:
@louiswol94 Databindings have fixed most of this PR. The version you tested had a no render_type(). But I did not see the heading tags update in Chromium as discussed. I've done some testing. Adding `render-type => template` back in has solved the tag update in the Dom when I'm in the editor. I've added it back in.